tests/scripts/all.sh

```diff
@@ -1335,6 +1341,173 @@ component_build_psa_want_ecdsa_disabled_software() {
     make CC=gcc CFLAGS="$ASAN_CFLAGS -DPSA_CRYPTO_DRIVER_TEST -DMBEDTLS_PSA_ACCEL_ALG_ECDSA -DMBEDTLS_PSA_ACCEL_ALG_DETERMINISTIC_ECDSA -I../tests/include -O2" LDFLAGS="$ASAN_CFLAGS"
 }
 
+# This should be renamed to test and updated once the accelerator ECDH code is in place and ready to test.
+component_build_psa_want_ecdh_disabled_software() {
```

Comment:
What about rather `component_build_psa_accel_alg_ecdh` as it is to test when ECDH computations are done by an accelerator? Positive definition instead of negative one. If you agree with this, ditto for the names of the other components.